python/servo/command_base.py

```diff
@@ -537,6 +537,9 @@ def build_android_env_if_needed(self, env: Dict[str, str]):
             if os.path.isdir(default):
                 env.setdefault(f"ANDROID_{kind.upper()}_ROOT", default)
 
+        if "IN_NIX_SHELL" and ("ANDROID_NDK_ROOT" not in env or "ANDROID_SDK_ROOT" not in env):
```

Comment:
Shouldn't this be `if "IN_NIX_SHELL" in env ` ?

Reply:
Yes; I fixed it.